publication-model/src/main/resources/publication-ontology.ttl

```diff
@@ -960,6 +960,82 @@ nva:DegreeLicentiate a rdfs:Class ;
   rdfs:isDefinedBy nva: ;
   rdfs:subClassOf nva:Degree .
 
+nva:AllowedFileOperation a rdfs:Class ;
+  rdfs:comment "A thing that indicates what you can do with a file" ;
+  rdfs:isDefinedBy nva: .
+
+nva:AllowedPublicationOperation a rdfs:Class ;
+  rdfs:comment "A thing that indicates what you can do with a publication" ;
+  rdfs:isDefinedBy nva: .
+
+nva:read_metadata a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates read access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:write_metadata a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates write access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:delete a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates delete access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:download a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates download access is allowed" ;
```

Comment:
the file may be downloaded

Reply:
Be careful, we should probably say "the" not "a"…if I wrote "a" elsewhere, sorry.